src/include/storage/store/column_chunk_data.h

```diff
@@ -198,9 +198,9 @@ class ColumnChunkData {
     virtual void setToInMemory();
     // numValues must be at least the number of values the ColumnChunk was first initialized
     // with
-    //reverse data and zero the part exceeding the original size
+    // reverse data and zero the part exceeding the original size
```

Comment:
Not sure why clang-format didn't do this in a previous PR

Reply:
It was a third-party contributor, so I guess that means the job didn't run (I guess it also got merged without my suggestion to correct it to `reserve` (it's not reversing any data).

Reply:
😞